Hestia/app/src/androidTest/java/com/rugged/application/hestia/UI/activities/login/LoginActivityTest.java

```diff
@@ -0,0 +1,4 @@
+package com.rugged.application.hestia.UI.activities.login;
+
+public class LoginActivityTest {
```

Comment:
All these empty classes.. How did this end up in your development branch? It should not have been approved.